next.config.js

```diff
@@ -1,59 +1,43 @@
-const { ANALYZE } = process.env;
-
-const withPWA = require("next-pwa");
-const withBundleAnalyzer = require("@next/bundle-analyzer")({
-  enabled: ANALYZE === "true",
+const prod = process.env.NODE_ENV === "production";
+const withPWA = require("next-pwa")({
+  dest: "public",
+  disable: !prod,
 });
 
 /** @type {import('next').NextConfig} */
 module.exports = withPWA({
-  pwa: {
```

Comment:
do we even use pwa here?

Reply:
we don't have all the functionalities of PWA but we have a badge in the lighthouse :) 
<img width="581" alt="image" src="https://user-images.githubusercontent.com/26671751/206903617-8c43e917-082f-4c8b-9047-c1af8ab123f3.png">